view_acl/__init__.py

```diff
@@ -0,0 +1,15 @@
+from django.core import urlresolvers
+from django.views.defaults import permission_denied
+
+from .utils import get_caller_name
+
+def autodiscover():
+    """
+    Block all view defined in urls.py by default.
+    Return 403 status, permission denied.
+    Skipped '/admin' and etc
+    """
+    url_patterns = get_caller_name()
```

Comment:
If we have name parameter specified to urlpatterns for the views we won't need to call get_caller_name(). The name will be used instead